### ISSUE TITLE
Fix duplication of side navigation

### ISSUE THREAD

**Issue by atsansone** [dart-lang/site-www#5297]
a11y script found duplicate left navigation data. This code exists in header.html and default.html. This raises multiple [WCAG F77 errors](https://squizlabs.github.io/HTML_CodeSniffer/Standards/WCAG2/#sc4_1_1).  This causes issues with accessibility technology. These lines should be hidden.

```html
          <div class="site-sidebar site-sidebar--header d-md-none">
            {% include_cached sidenav-level-1.html nav=site.data.sidenav page_url_path=route %}
          </div>
```

**Comment by parlough** [comment]
Unlike docs.flutter.dev, dart.dev only has one sidenav, so as far as I can tell this isn't a problem. Let me know if you find any more related issues though. Thanks!
